$OpenBSD: patch-plugins_todoist_gtd-provider-todoist_c,v 1.1 2017/11/01 21:19:21 ajacoutot Exp $

XXX push upstream
warning: implicit declaration of function 'strptime' is invalid in C99

Index: plugins/todoist/gtd-provider-todoist.c
--- plugins/todoist/gtd-provider-todoist.c.orig
+++ plugins/todoist/gtd-provider-todoist.c
@@ -16,7 +16,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#define _XOPEN_SOURCE
+#define _GNU_SOURCE 1
 
 #include "gtd-provider-todoist.h"
 #include "gtd-plugin-todoist.h"
